github-actions/verify-pr-title/src/index.ts

```diff
@@ -19,30 +21,43 @@ function run(): void {
 		// Examine the title for the expected patterns, which vary depending on what the target branch is.
 		const title = pullRequest.title as string;
 		const base = pullRequest.base as {ref: string};
-		const targetBranch = base.ref;
-		if (targetBranch === "release" || targetBranch === "documentation") {
-			// Release branches have their own less stringent format.
-			if (verifyReleaseBranchPrTitle(title)) {
-				console.log(`PR title '${title}' accepted for release branch.`);
+		const baseBranch = base.ref;
+		const head = pullRequest.head as {ref: string};
+		const headBranch = head.ref;
+		if (headBranch === "main" && baseBranch === "dev") {
+			// There's a title convention for merging `main` into `dev`.
+			if (verifyMain2DevPrTitle(title)) {
+				console.log(`PR title '${title}' accepted for dev branch.`);
 			} else {
 				core.setFailed(
-					`PR title '${title}' does not match the release PR template of "RELEASE: @W-XXXX@: Summary".`
+					`PR title '${title} does not match the template of "Main2Dev @W-XXXX@ Rebasing after vX.Y.Z"`
```

Comment:
Did you intend to drop the 2nd single quote after `'${title}`?

Reply:
Nope, that was a typo. Good catch.